go.mod

```diff
@@ -2,15 +2,20 @@ module github.com/gptscript-ai/go-gptscript
 
 go 1.23.0
 
-require github.com/getkin/kin-openapi v0.124.0
+require (
+	github.com/getkin/kin-openapi v0.124.0
+	github.com/stretchr/testify v1.8.4
```

Comment:
nit: I would prefer to not have this.

Reply:
nit: why? doesn't everyone use testify? What else would you use?